### ISSUE TITLE
Do not require contacts permission / drop that requirement as it scares users

### ISSUE THREAD

**Issue by rugk** [nextcloud/Android-SingleSignOn#771]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Feature request

**Which Nextcloud Version are you currently using:** 29

**Is your feature request related to a problem? Please describe.**
I tried to convince app maintainers to use this app, like in https://github.com/AntennaPod/AntennaPod/issues/7281

Unfortunately, the reply in https://github.com/AntennaPod/AntennaPod/issues/7281#issuecomment-2213066460 was:
> Signing in with the app requires adding the "read contacts" permission to AntennaPod. Users were concerned about that, so we decided against using it. See https://github.com/AntennaPod/AntennaPod/issues/6088

They have thus actively removed the integration of this lib, which is sad.

**Describe the solution you'd like**
Can you somehow just get rid of this permission? Even if it is optional, some users apparently dig there and complain because of that…

**Describe alternatives you've considered**
I know none…

Obviously not using the lib, but that is sad.

**Additional context**
May be solved(?) in or is at least related to https://github.com/nextcloud/Android-SingleSignOn/issues/49

**Comment by tobiasKaminsky** [comment]
There is no contacts needed. Only accounts, which is needed to display NC's accounts.

**Comment by rugk** [comment]
Yes, but AFAIK the issue is Android (or Google Play) shows this as the "contacts" permission to the user, which may scare them, as expxlained…

**Comment by tobiasKaminsky** [comment]
Totally understandable. But how can we change this? Write a text snippet, so that every app can show it to user in "details"?
I fear no one will read it…
